app/src/main/java/com/toggl/domain/AppState.kt

```diff
@@ -17,23 +14,11 @@ import com.toggl.models.domain.UserPreferences
 import com.toggl.models.domain.Workspace
 import com.toggl.onboarding.domain.states.OnboardingState
 import com.toggl.timer.common.domain.TimerState
-import java.time.DayOfWeek
 
 data class AppState(
     val user: Loadable<User> = Loadable.Uninitialized,
-    val userPreferences: UserPreferences = UserPreferences(
-        isManualModeEnabled = false,
-        is24HourClock = false,
-        selectedWorkspaceId = 1,
-        dateFormat = DateFormat.DDMMYYYY_dash,
-        durationFormat = DurationFormat.Improved,
-        firstDayOfTheWeek = DayOfWeek.MONDAY,
-        shouldGroupSimilarTimeEntries = true,
-        hasCellSwipeActions = true,
-        isCalendarIntegrationEnabled = false,
-        calendarIds = emptyList()
-    ),
-    val selectedSetting: SelectedSetting? = null,
+    val userPreferences: UserPreferences = UserPreferences.defaultUserPreferences,
```

Comment:
Why can't the defaults be part of the constructor so you can simply: 
```suggestion
    val userPreferences: UserPreferences = UserPreferences(),
```

Reply:
Two reasons for this:
1) I wanted to make this more obvious and create a situation where you either take all defaults or you have to provide everything.
2) I use these default values individually in `SettingsRepository::loadUserPreferences`